azext_iot/tests/iothub/test_iot_ext_int.py

```diff
@@ -40,8 +34,39 @@
 
 class TestIoTStorage(IoTLiveScenarioTest):
     def __init__(self, test_case):
-        super(TestIoTStorage, self).__init__(test_case, LIVE_HUB, LIVE_RG)
+
+        super(TestIoTStorage, self).__init__(test_case)
         self.managed_identity = None
+        if LIVE_STORAGE_ACCOUNT:
+            self.live_storage_uri = self.get_container_sas_url()
+
+    def get_container_sas_url(self):
+        from datetime import datetime, timedelta
+        from azure.storage.blob import ResourceTypes, AccountSasPermissions, generate_account_sas, BlobServiceClient
```

Comment:
Was there no combination of commands in the CLI able to obtain this value? For example `az storage container generate-sas`

Reply:
The SAS token generated using that command was not letting me to run the device-identity export command:

![image](https://user-images.githubusercontent.com/51140335/126010757-4bff6517-952f-4889-8103-9ae19e1d346e.png)

Hence, had to use this